src/main/java/dk/cngroup/lentils/entity/FinalPlace.java

```diff
@@ -27,21 +28,36 @@
     @NotNull(message = "Souřadnice nesmí být prázdné.")
     private Point location;
 
-    @Column(name = "opening_time")
+    @Column(name = "results_time")
     @DateTimeFormat(pattern = "yyyy-MM-dd'T'HH:mm")
-    private LocalDateTime openingTime;
+    private LocalDateTime resultsTime;
+
+    @Column(name = "access_time")
+    @NotNull(message = "Zpřístupnění nesmi být prázdné.")
```

Comment:
Tady to má jednu vadu. Díky tomu, že tady je podmínka NOT NULL a nemá to výchozí hodnotu, tak SQL příkaz pro přidání sloupce do tabulky při startu aplikace spadne (při pokusu při aktualizaci aplikace s předchozí verze).
Snaží se to totiž přidat sloupec bez hodnoty -> vloží se NULL, ale to je vlastně zakázané.

Reply:
v minulosti jsme vzdycky dropli databazi, ne? nejake napady jak na to ted?